### ISSUE TITLE
Standardise logo sizes

### ISSUE THREAD

**Issue by zkamvar** [carpentries/varnish#24]
at the moment, the SVGs of the logo size are all different (especially the incuabator and lab logos that I got from the template:

```
> imgs <- fs::dir_ls("inst/pkgdown/assets/assets/images")                                      
> img_df <- purrr::map(imgs, magick::image_read_svg) |> 
+   purrr::map_dfr(magick::image_info)     
> img_df$file <- fs::path_file(imgs)                                                           
> dplyr::filter(img_df, grepl("logo", file)) |> dplyr::arrange(width, height)                  
# A tibble: 12 × 8
   format width height colorspace matte filesize density file                   
   <chr>  <int>  <int> <chr>      <lgl>    <int> <chr>   <chr>                  
 1 PNG       53     23 sRGB       TRUE         0 72x72   software-logo-sm.svg   
 2 PNG       56     34 sRGB       TRUE         0 72x72   library-logo-sm.svg    
 3 PNG       77     79 sRGB       TRUE         0 72x72   carpentries-logo-sm.svg
 4 PNG       95     58 sRGB       TRUE         0 72x72   library-logo.svg       
 5 PNG      147     50 sRGB       TRUE         0 72x72   data-logo-sm.svg       
 6 PNG      147     50 sRGB       TRUE         0 72x72   data-logo.svg          
 7 PNG      225     46 sRGB       TRUE         0 72x72   software-logo.svg      
 8 PNG      238     50 sRGB       TRUE         0 72x72   carpentries-logo.svg   
 9 PNG      261    261 sRGB       TRUE         0 72x72   lab-logo-sm.svg        
10 PNG      261    261 sRGB       TRUE         0 72x72   lab-logo.svg           
11 PNG     1268   1630 sRGB       TRUE         0 72x72   incubator-logo-sm.svg  
12 PNG     1268   1630 sRGB       TRUE         0 72x72   incubator-logo.svg    
```

The small logos are for mobile views and the large logos are for web and we should probably just convert to favicons or standardize the two sizes.

**Comment by zkamvar** [comment]
note: this is related to #21

**Comment by zkamvar** [comment]
It turns out that the logo sizes were the same sizes as presented in the final figma drafts. The rule of thumb here (I think) is:

big logo: 50-80px because the containing div is 100px tall
small logo: 25-40px because the containing div is 50px tall

**Comment by zkamvar** [comment]
this was fixed in #30
